### PR TITLE
Include Eventer binary in Docker image

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @piosz

### CODE REVIEW

File: deploy/docker/Dockerfile [hunk 1]
```diff
@@ -9,4 +9,6 @@ RUN for cert in `ls -1 /etc/ssl/certs/*.crt | grep -v /etc/ssl/certs/ca-certific
 # cAdvisor discovery via external files.
 VOLUME /var/run/heapster/hosts
 ADD heapster /heapster
+ADD eventer /eventer
```
Comment (piosz) [inline deploy/docker/Dockerfile]: eventer is not started anywhere

Comment (mwielgus) [inline deploy/docker/Dockerfile]: Yes it is not, you can can specify what to start in yaml. By default Heapster4Metrics is started.

Comment (piosz) [inline deploy/docker/Dockerfile]: ack

**Comment by piosz** [comment]
LGTM

**Pull request merged**